sitemap.py

```diff
@@ -7,70 +7,72 @@
 from datetime import date
 
 # local
-from web_services import *
 import config
 
 import sentry_sdk
 from sentry_sdk import configure_scope
+from bu_cascade.cascade_connector import Cascade
 
 if config.SENTRY_URL:
     from sentry_sdk.integrations.flask import FlaskIntegration
     sentry_sdk.init(dsn=config.SENTRY_URL, integrations=[FlaskIntegration()])
 
 # todo Add logic for index pages
 
+cascade_connector = Cascade(config.SOAP_URL, config.CASCADE_LOGIN, config.SITE_ID, config.STAGING_DESTINATION_ID)
```

Comment:
cascade_connector might be a bad name. The file you are importing Cascade from is also called cascade_connector